### DISCUSSION TITLE
Broken display of labels with emoji

### DISCUSSION THREAD

**Discussion by 3c1u** [community/community#19186] [Projects and Issues] [answered]
Labels with emoji seem to be broken on my GitHub issue creation page.

![image](https://user-images.githubusercontent.com/298748/175067290-47d862f9-e1a9-452f-aa03-6699b59e3249.png)
![image](https://user-images.githubusercontent.com/298748/175068283-ca2c8edc-1041-4956-a110-4f160ca3d9d6.png)

**Comment by melvincwng** [comment, marked as answer]
@3c1u, you can report bugs to [Github Bug Report](https://support.github.com/contact/bug-report).

Hope this clarifies, thank you 😊
